docusaurus.config.js

```diff
@@ -2,10 +2,10 @@ module.exports = {
   title: 'SAP Cloud SDK',
   tagline:
     'A one stop shop for developing and extending SAP applications in a Cloud.',
-  url: 'https://github.com/SAP/cloud-sdk',
+  url: 'https://github.com/artemkovalyov/cloud-sdk',
```

Comment:
I guess this was for local testing too?

Reply:
Good catch, I shouldn't have committed this file.